deployment/ansible/roles/cac-tripplanner.app/defaults/main.yml

```diff
@@ -10,3 +10,13 @@ app_sass_version: "3.4.9"
 
 root_src_dir: "/opt/app/src"
 root_static_dir: "/srv/cac"
+
+cac_python_dependencies:
```

Comment:
It doesn't really hurt anything.

Also, though we strive not to, inevitably sometimes it is necessary to debug something on the server and having `django_extensions` and `ipython` there makes that easier.